### DISCUSSION TITLE
Is Draw IO the right tool for this job?

### DISCUSSION THREAD

**Discussion by trevorsmate** [jgraph/drawio#3605] [Q&A]
Got a client who wants me to digitise an existing manual spec form - below.

It essentially specifies pieces of fabric shown as a square (not to scale), which have standard elements placed around them - tabs, cutouts, velcro, splits. The overall height/width and position of the elements is then shown in figures. There are a small number of standard elements - maybe 10 but they need to be placed anywhere on the square representation in any orientation.

Hopefully to be used on a tablet with a stylus, list selection boxes for fabrics etc. , dates, dimension inputs, can I populate a customer list?

Any thoughts or suggestions gratefully received. Sean

[HeartLands - Linear .pdf](https://github.com/jgraph/drawio/files/11507379/HeartLands.-.Linear.pdf)

![form snapshots](https://github.com/jgraph/drawio/assets/133970687/8cd412cb-83ff-4746-a563-9702f24c0164)

..

**Comment by davidjgraph** [comment]
No, your requirements sound too bespoke for any off the shelf software.

**Reply by trevorsmate** [reply]
So if the budget doesn't stretch to a dedicated custom app (sounds expensive) what software could I bodge it in? I'm even thinking of dragging graphics around in a word doc as it has date inputs and drop down lists?
